contrib/for-tests/mount-tester/mt.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"syscall"
+)
+
+var (
+	fsTypePath           = ""
+	fileModePath         = ""
+	readFileContentPath  = ""
+	readWriteNewFilePath = ""
+)
+
+func init() {
+	flag.StringVar(&fsTypePath, "fs_type", "", "Path to print the fs type for")
+	flag.StringVar(&fileModePath, "file_mode", "", "Path to print the filemode of")
+	flag.StringVar(&readFileContentPath, "file_content", "", "Path to read the file content from")
+	flag.StringVar(&readWriteNewFilePath, "rw_new_file", "", "Path to kubeconfig containing embeded authinfo.")
```

Comment:
Bizarre, I do not know how that got in.  @jiangyaoguo, I will make you a PR to fix this nit.